apps/root/src/pages/position-detail/components/summary-container/components/position-data/index.tsx

```diff
@@ -279,465 +129,376 @@ const Details = ({ position, pendingTransaction, yieldOptions, totalGasSaved }:
   const foundYieldTo =
     position.to.underlyingTokens[0] && find(yieldOptions, { tokenAddress: position.to.underlyingTokens[0].address });
 
-  const toWithdrawToShow = showBreakdown ? toWithdrawBase : toWithdraw.amount;
-  const swappedToShow = showBreakdown ? swappedBase : swapped.amount;
-  const remainingLiquidityToShow = showBreakdown ? remainingLiquidity : totalRemainingLiquidity.amount;
-
   const executedSwaps = Number(totalSwaps) - Number(remainingSwaps);
 
   const isOldVersion = !VERSIONS_ALLOWED_MODIFY.includes(position.version);
 
   const isTestnet = TESTNETS.includes(positionNetwork.chainId);
 
+  const { mainCurrencyToken } = getNetworkCurrencyTokens(positionNetwork);
+
+  const positionStatusLabel = React.useMemo(() => {
```

Comment:
We could probably have this as a separate component right? so we declutter a bit the code from this one